### PR TITLE
[BREAKING CHANGE] support only interleaved_weights

### PR DESCRIPTION
**Pull request by mayank31398**

### CODE REVIEW

File: lm_engine/hf_models/modeling_utils/activations/glu.py [hunk 1]
```diff
@@ -39,24 +39,16 @@ def __init__(self, base_activation: nn.Module) -> GLUActivation:
...
-    def forward(self, x: torch.Tensor, is_interleaved: bool) -> torch.Tensor:
-        if (
-            is_kernel_allowed(Kernel.swiglu_packed)
-            and isinstance(self.base_activation, nn.SiLU)
-            and not is_interleaved
-        ):
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        if is_kernel_allowed(Kernel.swiglu_packed) and isinstance(self.base_activation, nn.SiLU):
+            # FIXME Mayank: fix this kernel in XMA to allow interleaved inputs
+            assert False
```
Comment (hanzlfs) [inline lm_engine/hf_models/modeling_utils/activations/glu.py]: just raise Unimplement error?

File: lm_engine/hf_models/modeling_utils/mlp_blocks/mlp.py [hunk 2]
```diff
@@ -106,52 +103,42 @@ def _fc1_act(self, x: torch.Tensor) -> torch.Tensor:
...
+            2 * up_weight.size(1),
+            *up_weight.size()[2:],
+            dtype=up_weight.dtype,
+            device=up_weight.device,
+        )
+        W[:, 1::2] = up_weight
+        W[:, ::2] = gate_weight
     else:
-        W = torch.cat([up_weight, gate_weight], dim=dim)
+        raise ValueError
```
Comment (hanzlfs) [inline lm_engine/hf_models/modeling_utils/mlp_blocks/mlp.py]: With some message? ValueError("dim >=2 is not supported xxx")
